packages/core/test/VDocument.test.ts

```diff
@@ -989,7 +1003,7 @@ describe('VDocument', () => {
                     await testEditor(BasicEditor, {
                         contentBefore: '<p class="a"><br></p><p>[]abc</p>',
                         stepFunction: deleteBackward,
-                        contentAfter: '<p class="a">[]abc</p>',
+                        contentAfter: '<p>[]abc</p>',
```

Comment:
Neither this one, obviously.

Reply:
But we just discussed it... Gspec...

Reply:
Well I did void my disagreement when we discussed it though. Discussing things does not mean agreeing on things you know... :p We just ended up changing the subject on this one.

Reply:
...